certified-connectors/Repfabric Lead Loader/Readme.md

```diff
@@ -0,0 +1,30 @@
+## Repfabric Lead Loader
+This connector is used for sending data to Repfabric's backend system, automatically creating new tables and objects related to customers, companies, and "leads". The primary purpose of this connector is to send the data from a .csv file, though this API can be used for other Microsoft services in the future.
+
+## Prerequisites
+You will only need access to a sheet template that will provide the correct data for the API to process, and a Power Automate template that uses the connector. [Here](https://repfabric-my.sharepoint.com/:x:/p/jeremy_ramos/EfdJI0HWs3lMi9T7rvNrGVsB83ZmKSciVHeOebrYulcTDw?e=1yafTd) is a Google Sheet example. Until we can publish the template with the certified connector, contact andre@phiquest.io for a Power Automate template.
+
+You will not need credentials for testing this operation as we cannot give credentials for this API to those outside of the organization at the moment. This API will use it own default credentials that are stored as Basic Auth, to avoid leaking client information.
+
+## Configuring the Connector
```

Comment:
@andreroberts1119 please review the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1), all of the required headings should be present.